src/components/Guides/Guides.js

```diff
@@ -4,7 +4,9 @@ import * as S from './styles'
 import Guide from './Guide'
 
 const Guides = props => {
-  const guides = props.guides.map(guide => <Guide {...guide} />)
+  console.log(props)
```

Comment:
Looks good, but let's remove the extra console log before pushing to prod if we're done with it.

Reply:
Should be fixed, thanks forgot about it :)